pkg/operator/configobserver/featuregates/observe_featuregates_test.go

```diff
@@ -66,7 +66,6 @@ func TestObserveFeatureFlags(t *testing.T) {
 				"RotateKubeletServerCertificate=true",
 				"DownwardAPIHugePages=true",
 				"PodSecurity=true",
-				"CSIDriverAzureFile=true",
```

Comment:
@soltysh @jsafrane can you check if this is the right fix for this test?

Reply:
Yes, the code is written in a way that the test must match openshift/api defaults [here](https://github.com/openshift/api/blob/441fe135b2fce3b4aac41a1488bea0b79cc6c800/config/v1/types_feature.go#L114)

(Some indirection would be nice though, feel free to fix it ;-) )